blocks/components/editable/index.js

```diff
@@ -127,6 +134,19 @@ export default class Editable extends wp.element.Component {
 		}
 	}
 
+	componentWillReceiveProps( nextProps ) {
+		const formatMap = {
+			strong: 'bold',
+			em: 'italic'
+		};
+
+		forEach( nextProps.formats, ( state, format ) => {
+			if ( state !== this.props.formats[ format ] ) {
+				this.editor.formatter.toggle( formatMap[ format ] );
```

Comment:
Thanks for the changes @iseulde. This starts to look good for me. But I think we should not use `toggle` here. I'm seeing some bugs while the format is changed when I click in different places of the Editable. I suspect this is caused by the `toggle` because we're not taking into account the value of `this.props.formats[ format ]` whether it's true or false. I don't know maybe something like `this.editor.execCommand( formatMap[ format ], true, this.props.formats[ format ] )` would fix it?